referencemodels/src/main/java/org/openehr/referencemodels/BuiltinReferenceModels.java

```diff
@@ -53,6 +53,8 @@ public static BmmRepository getBmmRepository() {
                 "bmm/openEHR/original/Release-1.0.2/openehr_ehr_102.bmm",
                 "bmm/openEHR/components/LANG/Release-1.0.0/openehr_lang_100.bmm",
                 "bmm/openEHR/components/RM/Release-1.0.4/openehr_rm_ehr_104.bmm",
+                "bmm/openEHR/components/PROC/Release-1.5.0/openehr_proc_task_planning_150.bmm",
+                "bmm/openEHR/components/PROC/Release-1.6.0/openehr_proc_task_planning_160.bmm",
```

Comment:
Do `bmm/openEHR/components/BASE/Release-1.0.4/openehr_base_104.bmm`, `bmm/openEHR/components/BASE/Release-1.0.4/openehr_expression_104.bmm` and/or the AM BMM files also need to be added here?

Reply:
It's possible. That is however a bigger change, and the AOM files are never needed for most archie users, who just want to validate archetypes against the BMM. And adding them would cost extra startup time. So, that could be a separate PR, with a lazy loading approach in case you need the AOM BMMs?